dts/bindings/serial/st,stm32-usart.yaml

```diff
@@ -20,3 +20,13 @@ properties:
 
         For example the USART1 would be
            pinctrl-0 = <&usart1_tx_pb6 &usart1_rx_pb7>;
+
+    stop-bits:
```

Comment:
Please also add this to `st,stm32-uart.yaml` and `st,stm32-lpuart.yaml` and we'll be fine